src/bug-filing/common/issue-filing-url-string-utils.ts

```diff
@@ -4,36 +4,35 @@ import { EnvironmentInfo } from '../../common/environment-info-provider';
 import { title } from '../../content/strings/application';
 import { CreateIssueDetailsTextData } from './../../common/types/create-issue-details-text-data';
 
-export class IssueFilingUrlStringUtils {
-    public static footer(environmentInfo: EnvironmentInfo): string {
+export type IssueUrlCreationUtils = {
+    footer: (info: EnvironmentInfo) => string;
+    collapseConsecutiveSpaces: (input: string) => string;
+    markdownEscapeBlock: (input: string) => string;
+    getSelectorLastPart: (selector: string) => string;
+    standardizeTags: (data: CreateIssueDetailsTextData) => string[];
+};
+
+export const IssueFilingUrlStringUtils: IssueUrlCreationUtils = {
```

Comment:
These are more about URL creation than strings. Perhaps `IssueCreationUrlUtils` would be a better name.

Reply:
That's the name of the type... words scrambled but same words...